sigs.yaml

```diff
@@ -5,6 +5,7 @@ sigs:
       Sample mission statement
     charter_link: sig-template/charter.md
     label: sig-template
```

Comment:
```suggestion
```

Reply:
Let's remove this for now and can revisit if we use labels like this in the future

Reply:
I'm a bit ambivalent either way, but I do think that if we use GitHub labels for everything in the SIG then it will make PRs & discussions easier to discover.

Maybe what this is missing is just the instruction that the value of this `label` field must be a label for the SIG created under https://github.com/cilium/cilium/issues/labels (or any other relevant org/repository)? When the SIG is created in this repo one of the maintainers responsible for merging the PR can ensure that the corresponding label exists.